### PR TITLE
Extend inline docs

### PR DESCRIPTION
**Pull request by payvint**

### CODE REVIEW

File: contracts/Wallets.sol [hunk 1]
```diff
@@ -95,16 +105,18 @@ contract Wallets is Permissions, IWallets {
...
+     * @dev Reimburse gas for node by validator wallet if node would have less than
+     * `minNodeBalance` amount after current tx. If validator wallet has not enough
+     * funds the node will receive the entire remaining amount in the validator's wallet.
      * `validatorId` - validator that will reimburse desired transaction
      * `spender` - address to send reimbursed funds
-     * `spentGas` - amount of spent gas that should be reimbursed to desired node
+     * `gasLimit` - amount of gas that should be reimbursed to desired node
      * 
-     * Emits a {NodeRefundedByValidator} event.
+     * Emits a {NodeRefundedByValidator} event if .
```
Comment (cstrangedk) [inline contracts/Wallets.sol]: remove "if"

File: contracts/Wallets.sol [hunk 2]
```diff
@@ -219,10 +233,18 @@ contract Wallets is Permissions, IWallets {
...
+     * @dev Returns schain eth balance.
+     * `schainHash` - keccak256 hash of schain name.
+     */
     function getSchainBalance(bytes32 schainHash) external view override returns (uint) {
         return _schainWallets[schainHash];
     }
 
+    /**
+     * @dev Returns validator eth balance.
+     * `validatorId` - id of existing validator.
```
Comment (cstrangedk) [inline contracts/Wallets.sol]: Please follow in-line doc standard

File: contracts/Wallets.sol [hunk 3]
```diff
@@ -245,7 +267,7 @@ contract Wallets is Permissions, IWallets {
 
     /**
      * @dev Recharge the schain wallet by schainHash (hash of schain name).
-     * `schainHash` - id of existing schain.
+     * `schainHash` - keccak256 hash of schain name..
```
Comment (cstrangedk) [inline contracts/Wallets.sol]: ibid.

File: contracts/Wallets.sol (hunk 1 above)
Comment (cstrangedk) [reply contracts/Wallets.sol]: and param descriptions to follow in-line doc standard

**Review by cstrangedk — APPROVED** [review]

**Review by yavrsky — APPROVED** [review]

**Pull request merged**